### PR TITLE
Using Whitenoise to serve static content

### PR DESCRIPTION
**Pull request by jon-betts**
This is nice for a few reasons:

 * Whitenoise automatically adds useful headers
 * It's more efficient serving content when it does happen
 * Reduces the code to test and maintain

This is for https://github.com/hypothesis/py-proxy/issues/31

----

Should be merged after: https://github.com/hypothesis/py-proxy/pull/53

### CODE REVIEW

File: py_proxy/static/index.html [hunk 1]
```diff
@@ -4,9 +4,9 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>via.hypothes.is</title>
-  <style>
-    @import url("//unpkg.com/normalize.css");
 
+  <link rel="stylesheet" href="/static/unpkg.com/normalize.css" />
```
Comment (seanh) [inline py_proxy/static/index.html]: Do you know if there's anything else that we were loading directly from third-party servers like this? If not, this change may fix https://github.com/hypothesis/via3/issues/38

File: py_proxy/views.py [hunk 2]
```diff
@@ -12,38 +11,12 @@
 CONFIG_FROM_FRAME = "via.request_config_from_frame"
 
 
-@view.view_config(renderer="py_proxy:templates/index.html.jinja2", route_name="index")
-def index(_request):
-    """Index endpoint."""
-    return {}
-
-
```
Comment (seanh) [inline py_proxy/views.py:20]: Sweet :+1:

File: py_proxy/views.py [hunk 3]
```diff
@@ -12,38 +11,12 @@
...
-@view.view_config(route_name="favicon", http_cache=(86400, {"public": True}))
-def favicon(request):
-    """Serve favicon.ico file."""
-    return response.FileResponse(
-        resource_filename("py_proxy", "static/favicon.ico"),
-        request=request,
-        content_type="image/x-icon",
-    )
-
-
```
Comment (seanh) [inline py_proxy/views.py:46]: :+1:

File: tests/unit/conftest.py [hunk 4]
```diff
@@ -7,7 +7,9 @@
 import pytest
 from pyramid import testing
 from pyramid.request import Request
+from webtest import TestApp as WebTestApp  # No pytest, this isn't a test class
```
Comment (seanh) [inline tests/unit/conftest.py]:
```suggestion
import webtest
```

...and use it as `webtest.TestApp`?

File: bin/update-pdfjs [hunk 5]
```diff
@@ -40,14 +40,5 @@ if [ $download_pdfjs ]; then
   # Remove example content from viewer
   rm $DEST_DIR/web/*.pdf
 
-  # Remove the check that the PDF being loaded is from the same origin as the
-  # viewer.
-  sed -i '' -e 's/HOSTED_VIEWER_ORIGINS.includes(viewerOrigin)/true \/* Hypothesis *\//' $DEST_DIR/web/viewer.js
+    echo $DEST_DIR/web/viewer.js
```
Comment (jon-betts) [inline bin/update-pdfjs:43]: I couldn't find this in viewer.js anymore. I suspect it's been removed by them.

**Review by seanh — APPROVED** [review]

File: py_proxy/static/index.html (hunk 1 above)
Comment (jon-betts) [reply py_proxy/static/index.html]: I think this is the only thing

File: tests/unit/conftest.py (hunk 4 above)
Comment (jon-betts) [reply tests/unit/conftest.py]: Done!

**Pull request merged**